mmv1/third_party/terraform/services/compute/resource_compute_network_test.go.tmpl

```diff
@@ -838,3 +872,17 @@ resource "google_compute_network" "acc_network_firewall_policy_enforcement_order
 }
 `, networkName, order)
 }
+
+func testAccComputeNetwork_resourceManagerTags(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_compute_network" "acc_network_with_resource_manager_tags" {
+  name                            = "%{network_name}"
+  auto_create_subnetworks         = false
+  params {
+	resource_manager_tags = {
+	  "%{tag_key_id}" = "%{tag_value_id}"
```

Comment:
It looks like you're making significant changes to the tag bootstrapping logic. What format will these values have? Is it significantly different than the other resources that have already added tag support?